crates/ruff_linter/src/rules/flake8_pytest_style/rules/raises.rs

```diff
@@ -26,8 +26,8 @@ use super::helpers::is_empty_or_null_string;
 ///
 /// def test_foo():
 ///     with pytest.raises(MyError):
-///         setup()  # may raise `MyError`
-///         func_to_test()
+///         setup()
+///         func_to_test()  # raises `MyError`
 ///         assert foo()  # not executed
```

Comment:
I think this proposed change defeats the point of the example.

In this example, the person writing the test wants to assert that `func_to_test` raises `MyError`. But the test they're writing has a bug in it -- the `setup()` function raises `MyError`, which means that the test passes even though the `pytest.raises()` block is exited before we even get to the function they wanted to test. The comment here is trying to point to the line that that should be moved out of the `pytest.raises()` block in order to avoid this antipattern

Reply:
From the example solution I got the impression that `setup()` cannot raise `MyError` because if it does then simply moving it out from the `with`-block is not the correct thing to do. But it was simple enough to address your feedback by making the comment more explicit.

Reply:
Yeah, I agree that it wasn't particularly clear as it was. I like the revised PR, thanks!